src/components/AppHeader.jsx

```diff
@@ -12,24 +19,40 @@ import { useSession } from '@inrupt/solid-ui-react';
 const AppHeader = () => {
   const { session } = useSession();
 
-  const headerStyle = {
-    width: '100%',
-    gap: '20px',
-    display: 'flex',
-    alignItems: 'center',
-    justifyContents: 'center'
-  };
-
   return (
-    <header style={headerStyle}>
-      <h2>Getting Started with PASS</h2>
-      {session.info.isLoggedIn ? (
-        <>
-          <Link to="/PASS/home/">Home</Link>
-          <Link to="/PASS/forms/">Forms</Link>
-        </>
-      ) : null}
-    </header>
+    <>
+      <CssBaseline />
+      <Box sx={{ flexGrow: 1 }}>
+        <AppBar position="static" color="primary">
+          <Toolbar>
+            {session.info.isLoggedIn ? (
+              <>
+                <Typography variant="h4" noWrap component="div">
+                  PASS
+                </Typography>
+                <ButtonGroup
+                  variant="contained"
+                  aria-label="outlined primary button group"
+                  color="tertiary"
+                  sx={{ margin: '0 0 0 20px' }}
+                >
+                  <Link to="/PASS/home/">
+                    <Button>Home</Button>
+                  </Link>
+                  <Link to="/PASS/forms/">
+                    <Button>Forms</Button>
+                  </Link>
+                </ButtonGroup>
+              </>
+            ) : (
+              <Typography variant="h4" noWrap component="div">
```

Comment:
I believe "Getting started with PASS" was centered in the newer version? I think that's fine. Of course if the screen gets too small, we could always shorted it to just "PASS".

We have need to consider using just PASS thought after login. Having PASS centered on mobile would be great (because of the small screen), but it might not work for laptop/desktop screens, since it'll look out of place. So we can make this responsive to be left-justified on mobile instead of centered. That would be great.

Reply:
Oh, I want to add. If we're wondering how to fit the links for mobile, I think we can use a dropdown hamburger menu for that, similar to what is shown in the newer version. But it should be grouped with the other icons.